File: iniciante/java/b1006.java
import java.util.Locale;
import java.util.Scanner;

public class b1006 {
  public static void main(String[] args) {
    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    double A = sc.nextDouble();
    double B = sc.nextDouble();
    double C = sc.nextDouble();
    double media = ((A * 2) + (B * 3) + (C * 5)) / 10.0;

    System.out.println("MEDIA = " + String.format("%.1f", media));

    sc.close();
  }
}

File: iniciante/swift/b1006.swift
import Foundation

let a = Double(readLine(strippingNewline: true)!)!
let b = Double(readLine(strippingNewline: true)!)!
let c = Double(readLine(strippingNewline: true)!)!

var media = ((a * 2) + (b * 3) + (c * 5)) / 10.0
print(String(format:"MEDIA = %.1f", media))